assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
```

Comment:
You should nest the nest the style rules if that's how you want the links to be used. People will probably want both, default styles to work for most situations and ways to override them with classes.

I still stand by what I said before though. As a developer, when I see an `elements` folder, I would suspect only rules for html elements, a, h1, etc, would be in there, no classes. So I would have trouble finding where the `usa-link` class would be if I was working on the code base. It would also help for specificity organization that classes don't come until later in the import order in the final file to ensure you have low specificity first so don't override unwanted things.

I think you have to make the best decision on the research you guys did. Just be careful with specificity problems, that's like one of the only things in CSS architecture that really matters.